.gitattributes

```diff
@@ -1,5 +1,5 @@
 # Define the line ending behavior of the different file extensions
-# Set default behaviour, in case users don't have core.autocrlf set.
+# Set default behavior, in case users don't have core.autocrlf set.
```

Comment:
Her royal majesty the Queen wouldn't like this one. ;)

Reply:
Good thing you made yourself independent from them long time ago.. ^^